service/history/transferQueueStandbyTaskExecutor.go

```diff
@@ -459,7 +459,7 @@ func (t *transferQueueStandbyTaskExecutor) processRecordWorkflowStartedOrUpsertH
 	startTime := timestamp.TimeValue(startEvent.GetEventTime())
 	executionTimestamp := getWorkflowExecutionTime(mutableState, startEvent)
 	visibilityMemo := getWorkflowMemo(executionInfo.Memo)
-	searchAttr := getSearchAttributes(copySearchAttributes(executionInfo.SearchAttributes))
+	searchAttr := getSearchAttributes(executionInfo.SearchAttributes)
```

Comment:
I'm assuming the entire function is called under mutableState lock?

Reply:
yes